src/CSharp/EasyMicroservice.MemoryPack/Sample.cs

```diff
@@ -0,0 +1,31 @@
+﻿using MemoryPack;
```

Comment:
Please add your sample file to the test cases.

Reply:
@AminMatini can you please add this file to the test cases and add test cases for this project?